Makefile

```diff
@@ -237,7 +246,8 @@ version:
 
 test: $(GOBUILDER) | $(DIST)
 	@echo Running tests on $(GOMODULE)
-	@$(DOCKER_GO) "gotestsum --jsonfile $(DOCKER_DIST)/results.json --junitfile $(DOCKER_DIST)/results.xml" $(GOTREE) $(GOMODULE)
+	$(QUIET)$(DOCKER_GO) "gotestsum --jsonfile $(DOCKER_DIST)/results.json --junitfile $(DOCKER_DIST)/results.xml" $(GOTREE) $(GOMODULE)
+	: $@: Succeeded
```

Comment:
Quick question: what's the point of all these:

```
: $@: Succeeded
```

They seem to add to the noise and I can't figure out how to NOT see them

Reply:
You could complain that it adds too much noise if this was one of those finely tuned Makefiles that only generates a 1-line summary for each step, but this Makefile the other extreme that generates tons of output, especially from the docker commands. The worse thing for debugging build errors is not know which mystery, hidden command is generating the output. You need as much help as possible to map it back to the make step it is running. These Succeeded lines make it so easy to pinpoint where it is in the make flow. But you could run make -s if you really hate them.